Remove duplicate `webaudio` local tests in favor of WPT

Ahmad-S792 opened the pull request:
#### ecd990add34621312cda6e9edabf7ec757a04dbd
<pre>
Remove duplicate `webaudio` local tests in favor of WPT

<a href="https://bugs.webkit.org/show_bug.cgi?id=279865">https://bugs.webkit.org/show_bug.cgi?id=279865</a>
<a href="https://rdar.apple.com/136198407">rdar://136198407</a>

Reviewed by Youenn Fablet.

This patch is to remove local `webaudio` tests in favor of WPT, which are
also in tree.

These tests exist in `webaudio` path on WPT and they were uploaded to WPT
by Blink / Google in below referred issue [1]:

[1] <a href="https://issues.chromium.org/issues/40530609">https://issues.chromium.org/issues/40530609</a>

&gt; audiobuffer.html -&gt; the-audiobuffer-interface/audiobuffer.html
&gt; audiobuffersource-channels.html -&gt; the-audiobuffersourcenode-interface/audiobuffersource-channels.html
&gt; audiobuffersource-ended.html -&gt; the-audiobuffersourcenode-interface/audiobuffersource-ended.html
&gt; audiobuffersource-multi-channels.html -&gt; the-audiobuffersourcenode-interface/audiobuffersource-multi-channels.html
&gt; audiobuffersource-start.html -&gt; the-audiobuffersourcenode-interface/audiobuffersource-start.html
&gt; audiochannelmerger-basic.html -&gt; the-channelmergernode-interface/audiochannelmerger-basic.html
&gt; audiochannelsplitter.html -&gt; the-channelsplitternode-interface/audiochannelsplitter.html
&gt; audionode-connect-order.html -&gt; the-audionode-interface/audionode-connect-order.html
&gt; audionode.html -&gt; the-audionode-interface/audionode.html
&gt; audioparam-connect-audioratesignal.html -&gt; the-audioparam-interface/audioparam-connect-audioratesignal.html
&gt; audioparam-setValueAtTime.html -&gt; the-audioparam-interface/audioparam-setValueAtTime.html
&gt; audioparam-summingjunction.html -&gt; the-audioparam-interface/audioparam-summingjunction.html
&gt; convolver-channels.html -&gt; the-convolvernode-interface/convolver-channels.html
&gt; delaynode-max-default-delay.html -&gt; the-delaynode-interface/delaynode-max-default-delay.html
&gt; delaynode.html -&gt; the-delaynode-interface/delaynode.html
&gt; delaynode-scheduling.html -&gt; the-delaynode-interface/delaynode-scheduling.html
&gt; delaynode-maxdelaylimit.html -&gt; the-delaynode-interface/delaynode-maxdelaylimit.html
&gt; delaynode-max-nondefault-delay.html -&gt; the-delaynode-interface/delaynode-max-nondefault-delay.html
&gt; delaynode-maxdelay.html -&gt; the-delaynode-interface/delaynode-maxdelay.html
&gt; distance-exponential.html -&gt; the-pannernode-interface/distance-exponential.html
&gt; distance-inverse.html -&gt; the-pannernode-interface/distance-inverse.html
&gt; distance-linear.html -&gt; the-pannernode-interface/distance-linear.html
&gt; dynamicscompressor-basic.html -&gt; the-dynamicscompressornode-interface/dynamicscompressor-basic.html
&gt; gain-basic.html -&gt; the-gainnode-interface/gain-basic.html
&gt; gain.html -&gt; the-gainnode-interface/gain.html
&gt; pannernode-basic.html -&gt; the-stereopanner-interface/stereopannernode-basic.html
&gt; realtimeanalyser-fft-sizing.html -&gt; the-analysernode-interface/realtimeanalyser-fft-sizing.html
&gt; sample-accurate-scheduling.html -&gt; the-audiobuffersourcenode-interface/sample-accurate-scheduling.html
&gt; waveshaper.html -&gt; the-waveshapernode-interface/waveshaper.html

* LayoutTests/webaudio/audiobuffer-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffer.html: Removed.
* LayoutTests/webaudio/audiobuffersource-channels-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-channels.html: Removed.
* LayoutTests/webaudio/audiobuffersource-ended-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-ended.html: Removed.
* LayoutTests/webaudio/audiobuffersource-multi-channels-expected.wav: Removed.
* LayoutTests/webaudio/audiobuffersource-multi-channels.html: Removed.
* LayoutTests/webaudio/audiobuffersource-start-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-start.html: Removed.
* LayoutTests/webaudio/audiochannelmerger-basic-expected.txt: Removed.
* LayoutTests/webaudio/audiochannelmerger-basic.html: Removed.
* LayoutTests/webaudio/audiochannelsplitter-expected.txt: Removed.
* LayoutTests/webaudio/audiochannelsplitter.html: Removed.
* LayoutTests/webaudio/audionode-connect-order-expected.txt: Removed.
* LayoutTests/webaudio/audionode-connect-order.html: Removed.
* LayoutTests/webaudio/audionode-expected.txt: Removed.
* LayoutTests/webaudio/audionode.html: Removed.
* LayoutTests/webaudio/audioparam-connect-audioratesignal-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-connect-audioratesignal.html: Removed.
* LayoutTests/webaudio/audioparam-setValueAtTime-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-setValueAtTime.html: Removed.
* LayoutTests/webaudio/audioparam-summingjunction-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-summingjunction.html: Removed.
* LayoutTests/webaudio/convolver-channels-expected.txt: Removed.
* LayoutTests/webaudio/convolver-channels.html: Removed.
* LayoutTests/webaudio/delaynode-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-default-delay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-default-delay.html: Removed.
* LayoutTests/webaudio/delaynode-max-nondefault-delay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-nondefault-delay.html: Removed.
* LayoutTests/webaudio/delaynode-maxdelay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-maxdelay.html: Removed.
* LayoutTests/webaudio/delaynode-maxdelaylimit-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-maxdelaylimit.html: Removed.
* LayoutTests/webaudio/delaynode-scheduling-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-scheduling.html: Removed.
* LayoutTests/webaudio/delaynode.html: Removed.
* LayoutTests/webaudio/distance-exponential-expected.txt: Removed.
* LayoutTests/webaudio/distance-exponential.html: Removed.
* LayoutTests/webaudio/distance-inverse-expected.txt: Removed.
* LayoutTests/webaudio/distance-inverse.html: Removed.
* LayoutTests/webaudio/distance-linear-expected.txt: Removed.
* LayoutTests/webaudio/distance-linear.html: Removed.
* LayoutTests/webaudio/dynamicscompressor-basic-expected.txt: Removed.
* LayoutTests/webaudio/dynamicscompressor-basic.html: Removed.
* LayoutTests/webaudio/gain-basic-expected.txt: Removed.
* LayoutTests/webaudio/gain-basic.html: Removed.
* LayoutTests/webaudio/gain.html: Removed.
* LayoutTests/webaudio/pannernode-basic-expected.txt: Removed.
* LayoutTests/webaudio/pannernode-basic.html: Removed.
* LayoutTests/webaudio/realtimeanalyser-fft-sizing-expected.txt: Removed.
* LayoutTests/webaudio/realtimeanalyser-fft-sizing.html: Removed.
* LayoutTests/webaudio/sample-accurate-scheduling-expected.txt: Removed.
* LayoutTests/webaudio/sample-accurate-scheduling.html: Removed.
* LayoutTests/webaudio/waveshaper-expected.txt: Removed.
* LayoutTests/webaudio/waveshaper.html: Removed.

Canonical link: <a href="https://commits.webkit.org/283882@main">https://commits.webkit.org/283882@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/80d6cde4796f362a9f7bc187e8801071d5eea29d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/67617 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/46996 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/20256 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/71670 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/18754 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/54794 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/18558 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/54150 "Passed tests") | [![loading-orange](https://github-production-user-asset-6210df.s3.amazonaws.com/3098702/291015173-08c448be-ac0a-4fd6-92a3-8165057445b7.png) 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/12543 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Skipped layout-tests; layout-tests running") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/70684 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/43128 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/58482 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/34620 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/39801 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/15887 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/17116 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/61760 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/16229 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/73370 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/11576 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/15538 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/61597 "Found 1 new test failure: media/video-seek-have-nothing.html (failure)") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/11611 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/58550 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/61650 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/15031 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/9455 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/3074 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/42802 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/43880 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/45069 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/43619 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

youennf reviewed (approved):
Let's add the link to the Chrome bug that did the conversion and the upload to WPT in the commit message.

Ahmad-S792 committed:
Remove duplicate `webaudio` local tests in favor of WPT

https://bugs.webkit.org/show_bug.cgi?id=279865
rdar://136198407

Reviewed by Youenn Fablet.

This patch is to remove local `webaudio` tests in favor of WPT, which are
also in tree.

These tests exist in `webaudio` path on WPT and they were uploaded to WPT
by Blink / Google in below referred issue [1]:

[1] https://issues.chromium.org/issues/40530609

> audiobuffer.html -> the-audiobuffer-interface/audiobuffer.html
> audiobuffersource-channels.html -> the-audiobuffersourcenode-interface/audiobuffersource-channels.html
> audiobuffersource-ended.html -> the-audiobuffersourcenode-interface/audiobuffersource-ended.html
> audiobuffersource-multi-channels.html -> the-audiobuffersourcenode-interface/audiobuffersource-multi-channels.html
> audiobuffersource-start.html -> the-audiobuffersourcenode-interface/audiobuffersource-start.html
> audiochannelmerger-basic.html -> the-channelmergernode-interface/audiochannelmerger-basic.html
> audiochannelsplitter.html -> the-channelsplitternode-interface/audiochannelsplitter.html
> audionode-connect-order.html -> the-audionode-interface/audionode-connect-order.html
> audionode.html -> the-audionode-interface/audionode.html
> audioparam-connect-audioratesignal.html -> the-audioparam-interface/audioparam-connect-audioratesignal.html
> audioparam-setValueAtTime.html -> the-audioparam-interface/audioparam-setValueAtTime.html
> audioparam-summingjunction.html -> the-audioparam-interface/audioparam-summingjunction.html
> convolver-channels.html -> the-convolvernode-interface/convolver-channels.html
> delaynode-max-default-delay.html -> the-delaynode-interface/delaynode-max-default-delay.html
> delaynode.html -> the-delaynode-interface/delaynode.html
> delaynode-scheduling.html -> the-delaynode-interface/delaynode-scheduling.html
> delaynode-maxdelaylimit.html -> the-delaynode-interface/delaynode-maxdelaylimit.html
> delaynode-max-nondefault-delay.html -> the-delaynode-interface/delaynode-max-nondefault-delay.html
> delaynode-maxdelay.html -> the-delaynode-interface/delaynode-maxdelay.html
> distance-exponential.html -> the-pannernode-interface/distance-exponential.html
> distance-inverse.html -> the-pannernode-interface/distance-inverse.html
> distance-linear.html -> the-pannernode-interface/distance-linear.html
> dynamicscompressor-basic.html -> the-dynamicscompressornode-interface/dynamicscompressor-basic.html
> gain-basic.html -> the-gainnode-interface/gain-basic.html
> gain.html -> the-gainnode-interface/gain.html
> pannernode-basic.html -> the-stereopanner-interface/stereopannernode-basic.html
> realtimeanalyser-fft-sizing.html -> the-analysernode-interface/realtimeanalyser-fft-sizing.html
> sample-accurate-scheduling.html -> the-audiobuffersourcenode-interface/sample-accurate-scheduling.html
> waveshaper.html -> the-waveshapernode-interface/waveshaper.html

* LayoutTests/webaudio/audiobuffer-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffer.html: Removed.
* LayoutTests/webaudio/audiobuffersource-channels-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-channels.html: Removed.
* LayoutTests/webaudio/audiobuffersource-ended-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-ended.html: Removed.
* LayoutTests/webaudio/audiobuffersource-multi-channels-expected.wav: Removed.
* LayoutTests/webaudio/audiobuffersource-multi-channels.html: Removed.
* LayoutTests/webaudio/audiobuffersource-start-expected.txt: Removed.
* LayoutTests/webaudio/audiobuffersource-start.html: Removed.
* LayoutTests/webaudio/audiochannelmerger-basic-expected.txt: Removed.
* LayoutTests/webaudio/audiochannelmerger-basic.html: Removed.
* LayoutTests/webaudio/audiochannelsplitter-expected.txt: Removed.
* LayoutTests/webaudio/audiochannelsplitter.html: Removed.
* LayoutTests/webaudio/audionode-connect-order-expected.txt: Removed.
* LayoutTests/webaudio/audionode-connect-order.html: Removed.
* LayoutTests/webaudio/audionode-expected.txt: Removed.
* LayoutTests/webaudio/audionode.html: Removed.
* LayoutTests/webaudio/audioparam-connect-audioratesignal-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-connect-audioratesignal.html: Removed.
* LayoutTests/webaudio/audioparam-setValueAtTime-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-setValueAtTime.html: Removed.
* LayoutTests/webaudio/audioparam-summingjunction-expected.txt: Removed.
* LayoutTests/webaudio/audioparam-summingjunction.html: Removed.
* LayoutTests/webaudio/convolver-channels-expected.txt: Removed.
* LayoutTests/webaudio/convolver-channels.html: Removed.
* LayoutTests/webaudio/delaynode-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-default-delay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-default-delay.html: Removed.
* LayoutTests/webaudio/delaynode-max-nondefault-delay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-max-nondefault-delay.html: Removed.
* LayoutTests/webaudio/delaynode-maxdelay-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-maxdelay.html: Removed.
* LayoutTests/webaudio/delaynode-maxdelaylimit-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-maxdelaylimit.html: Removed.
* LayoutTests/webaudio/delaynode-scheduling-expected.txt: Removed.
* LayoutTests/webaudio/delaynode-scheduling.html: Removed.
* LayoutTests/webaudio/delaynode.html: Removed.
* LayoutTests/webaudio/distance-exponential-expected.txt: Removed.
* LayoutTests/webaudio/distance-exponential.html: Removed.
* LayoutTests/webaudio/distance-inverse-expected.txt: Removed.
* LayoutTests/webaudio/distance-inverse.html: Removed.
* LayoutTests/webaudio/distance-linear-expected.txt: Removed.
* LayoutTests/webaudio/distance-linear.html: Removed.
* LayoutTests/webaudio/dynamicscompressor-basic-expected.txt: Removed.
* LayoutTests/webaudio/dynamicscompressor-basic.html: Removed.
* LayoutTests/webaudio/gain-basic-expected.txt: Removed.
* LayoutTests/webaudio/gain-basic.html: Removed.
* LayoutTests/webaudio/gain.html: Removed.
* LayoutTests/webaudio/pannernode-basic-expected.txt: Removed.
* LayoutTests/webaudio/pannernode-basic.html: Removed.
* LayoutTests/webaudio/realtimeanalyser-fft-sizing-expected.txt: Removed.
* LayoutTests/webaudio/realtimeanalyser-fft-sizing.html: Removed.
* LayoutTests/webaudio/sample-accurate-scheduling-expected.txt: Removed.
* LayoutTests/webaudio/sample-accurate-scheduling.html: Removed.
* LayoutTests/webaudio/waveshaper-expected.txt: Removed.
* LayoutTests/webaudio/waveshaper.html: Removed.

Canonical link: https://commits.webkit.org/283882@main
